ammo check/gamedata/scripts/ammo_check_mcm.script

```diff
@@ -8,17 +8,17 @@ local jammed_weapon = false
 
 -- the miss alighnment of the colors and the text 2x the info without 2x the text. change it if it's weird. 
 local messages = {
-    {m = "st_ac_near_empty", c = utils_xml.get_color("d_red", true)}, -- <1/10
-    {m = "st_ac_near_empty", c = utils_xml.get_color("red", true)}, -- <2/10
-    {m = "st_ac_less_half", c = utils_xml.get_color("red", true)}, -- <3/10
-    {m = "st_ac_less_half", c = utils_xml.get_color("orange", true)}, -- <4/10
-    {m = "st_ac_about_half", c = utils_xml.get_color("orange", true)}, -- <5/10
-    {m = "st_ac_about_half", c = utils_xml.get_color("yellow", true)}, -- <6/10
-    {m = "st_ac_more_half", c = utils_xml.get_color("yellow", true)}, -- <7/10
-    {m = "st_ac_more_half", c = utils_xml.get_color("green", true)}, -- <8/10
-    {m = "st_ac_nearly_full", c = utils_xml.get_color("green", true)}, -- <9/10
-    {m = "st_ac_nearly_full", c = utils_xml.get_color("white", true)}, -- <10/10
-    {m = "st_ac_full", c = utils_xml.get_color("white", true)} -- full
+    {m = "st_ac_near_empty", c = GetARGB(0xff, 0xff, 0x40, 0x00)}, -- <1/10, Red Orange
```

Comment:
I suggest changing this one to the red, shifting all the other colors down one and only have one laser lemon. If the text and colors change out of step that doubles the granularity of the information being provided without having too many words or colors. Probably only useful for really large mags but i like the idea.

Reply:
I thought about it too. Adding extra information subtly via color sounds good. In the end, I just went for consistency and made a 1-1 mapping with colors and messages. However, I don't have strong feelings one way or the other.

I'll test with your suggestion and see how it looks in practice. If it's not jarring or confusing, I'll go with granularity.

Reply:
Yes, it doesn't look half bad. Added the changes.

Jam messages is now red too.